kubernetes/loculus/values.yaml

```diff
@@ -1529,7 +1529,9 @@ secrets:
       username: "dummy"
       password: "dummy"
 runDevelopmentKeycloakDatabase: true
+runDevelopmentKeycloakDatabaseWithPersistence: true
```

Comment:
IMO `run` is confusing naming here - we've used it above to mean turn on the database. I would guess that I would only need to set `runDevelopmentMainDatabaseWithPersistence` and not `runDevelopmentMainDatabase` (we I think intend to set `runDevelopmentMainDatabase`, etc. to default to `false` at some point).

Reply:
For me the ideal would be `persistentDevDatabases` which would apply to both as I don't really see a reason one would want to have one persistent but not the other

Reply:
or `devDatabasePersistence`

Reply:
For development one will always want it to be cluster-internal and not externally managed?

Yeah for future would be better to make this enum, something like:

```
keycloakDatabase: 'ephemeral' | 'semi-persistent' | 'persistent' | 'external'
```

Feel free to change it to whatever, in my view we shouldn't waste too much thought on such dev-only features for now. Happy to rename anytime.

Reply:
ok it's `developmentDatabasePersistence: true` for all